UIKitTask/Sources/Scenes/BillScene/BillViewController.swift

```diff
@@ -0,0 +1,199 @@
+// BillViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+protocol BillViewControllerProtocol: AnyObject {
+    func didClosedScreen()
+}
+
+/// Экран для показа итоговой суммы продукта и перехода для оплаты
+final class BillViewController: UIViewController {
+    // MARK: - Visual Components
+
+    private let whiteView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white
+        view.layer.cornerRadius = 20
+        return view
+    }()
+
+    private lazy var closeButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(named: "x"), for: .normal)
+        button.addTarget(self, action: #selector(closeButtonTapped), for: .touchUpInside)
+        return button
+    }()
+
+    private let leftBranchImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFit
+        imageView.image = UIImage(named: "left branch")
+        return imageView
+    }()
+
+    private let rightBranchImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFit
+        imageView.image = UIImage(named: "right branch")
+        return imageView
+    }()
+
+    private let orderLabel: UILabel = {
+        let label = UILabel()
+        label.font = .setVerdanaBold(withSize: 18)
+        label.textColor = .black
+        label.textAlignment = .center
+        label.text = "Вашъ Заказъ"
```

Comment:
```suggestion
        label.text = ""
```
В константы
и ниже тоже